core/block/object/objectgraph/graph.go

```diff
@@ -64,75 +66,78 @@ func (gr *Builder) Name() (name string) {
 }
 
 func (gr *Builder) ObjectGraph(req *pb.RpcObjectGraphRequest) ([]*types.Struct, []*pb.RpcObjectGraphEdge, error) {
-	records, err := gr.queryRecords(req)
+	relations, err := gr.objectStore.ListAllRelations(req.SpaceId)
+	if err != nil {
+		return nil, nil, err
+	}
+	req.Keys = append(req.Keys, bundle.RelationKeyLinks.String())
+	req.Keys = append(req.Keys, lo.FilterMap(relations, func(rel *relationutils.Relation, _ int) (string, bool) {
+		return rel.Key, isRelationShouldBeIncludedAsEdge(rel)
+	})...)
+
+	resp, err := gr.subscriptionService.Search(pb.RpcObjectSearchSubscribeRequest{
+		Source:       req.SetSource,
+		Filters:      req.Filters,
+		Keys:         lo.Map(relations.Models(), func(rel *model.Relation, _ int) string { return rel.Key }),
+		CollectionId: req.CollectionId,
+		Limit:        int64(req.Limit),
+	})
+
 	if err != nil {
 		return nil, nil, err
 	}
 
+	err = gr.subscriptionService.Unsubscribe(resp.SubId)
```

Comment:
add comment please that it's a temporary workaround and we should use another method for Search without creating the subscription

Reply:
thanks added to the task